packages/fxa-settings/src/components/FormVerifyTotp/index.tsx

```diff
@@ -2,87 +2,101 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { useRef, useState } from 'react';
-import TotpInputGroup from '../TotpInputGroup';
-import { FtlMsg } from 'fxa-react/lib/utils';
-import Banner, { BannerType } from '../Banner';
+import React, { useState } from 'react';
+import InputText from '../InputText';
+import { useForm } from 'react-hook-form';
+import { getLocalizedErrorMessage } from '../../lib/error-utils';
+import { useFtlMsgResolver } from '../../models';
+import { AuthUiErrors } from '../../lib/auth-errors/auth-errors';
+import { FormVerifyTotpProps, VerifyTotpFormData } from './interfaces';
 
-export type CodeArray = Array<string | undefined>;
-
-export type FormVerifyTotpProps = {
-  codeLength: 6 | 8;
-  errorMessage: string;
-  localizedInputGroupLabel: string;
-  localizedSubmitButtonText: string;
-  setErrorMessage: React.Dispatch<React.SetStateAction<string>>;
-  verifyCode: (code: string) => void;
-};
+// Split inputs are not recommended for accesibility
+// Code inputs should be a single input field
+// See FXA-10390 for details on reverting from split input to single input
```

Comment:
It would be really nice to A/B test this on the password reset flow so that we have some clear statistics on the performance of single input vs multipile input fields.

Reply:
Agree, would be nice to know if there were other measurable impacts of switching to split input component, but I don't think we'll be going back considering the accessibility issues. Maybe we can revisit with a small sample size once we have Nimbus integrated.